### PR TITLE
Creates `transition_risk_category` and `transition_risk_category_share` at company level for transition risk profile indicator

### LINKED ISSUE #256: Create transition risk categories on company level
Dear @kalashsinghal, similiar to the emission profile and sector profile, we also need transition risk categories on company level. That means we need to aggregate the product level results also on company level. I think at company level the columns are called: `emission_category`, `sector_category`.

### PR DESCRIPTION
**Pull request by kalashsinghal**
closes #256 

Dear @AnneSchoenauer @Tilmon @maurolepore 

This PR creates `transition_risk_category` and `transition_risk_category_share` columns at company level for transition risk profile indicator. Please review the output using this reprex:

``` r
library(readr, warn.conflicts = FALSE)
library(dplyr, warn.conflicts = FALSE)
library(tiltToyData, warn.conflicts = FALSE)
library(tiltTransitionRisk, warn.conflicts = FALSE)
devtools::load_all(".")
#> ℹ Loading tiltIndicatorAfter
options(width = 5000)

set.seed(123)
restore <- options(list(
  readr.show_col_types = FALSE,
  tiltIndicatorAfter.output_co2_footprint = TRUE
))

toy_emissions_profile_products_ecoinvent <- read_csv(toy_emissions_profile_products_ecoinvent())
toy_emissions_profile_any_companies <- read_csv(toy_emissions_profile_any_companies()) |>
  mutate(companies_id = case_when(
    companies_id == "antimonarchy_canine" ~ "nonphilosophical_llama",
    TRUE ~ companies_id
  ))
toy_sector_profile_any_scenarios <- read_csv(toy_sector_profile_any_scenarios())
toy_sector_profile_companies <- read_csv(toy_sector_profile_companies()) |>
  mutate(companies_id = case_when(
    companies_id == "antimonarchy_canine" ~ "nonphilosophical_llama",
    TRUE ~ companies_id
  ))
toy_europages_companies <- read_csv(toy_europages_companies())
toy_ecoinvent_activities <- read_csv(toy_ecoinvent_activities())
toy_ecoinvent_europages <- read_csv(toy_ecoinvent_europages())
toy_ecoinvent_inputs <- read_csv(toy_ecoinvent_inputs())
toy_isic_name <- read_csv(toy_isic_name())
toy_all_activities_scenario_sectors <- read_csv(toy_all_activities_scenario_sectors())

toy_emissions_profile <- profile_emissions(
  companies = toy_emissions_profile_any_companies,
  co2 = toy_emissions_profile_products_ecoinvent,
  europages_companies = toy_europages_companies,
  ecoinvent_activities = toy_ecoinvent_activities,
  ecoinvent_europages = toy_ecoinvent_europages,
  isic = toy_isic_name
)

toy_sector_profile <- profile_sector(
  companies = toy_sector_profile_companies,
  scenarios = toy_sector_profile_any_scenarios,
  europages_companies = toy_europages_companies,
  ecoinvent_activities = toy_ecoinvent_activities,
  ecoinvent_europages = toy_ecoinvent_europages,
  isic = toy_isic_name
)

output <- transition_risk_profile(
  emissions_profile = toy_emissions_profile,
  sector_profile = toy_sector_profile,
  co2 = toy_emissions_profile_products_ecoinvent,
  all_activities_scenario_sectors = toy_all_activities_scenario_sectors,
  scenarios = toy_sector_profile_any_scenarios,
  pivot_wider = FALSE
)

product <- output |> 
  unnest_product() |>
  filter(companies_id == "nonphilosophical_llama",
         benchmark_tr_score %in% c("1.5C RPS_2030_all", "1.5C RPS_2030_isic_4digit", "1.5C RPS_2030_tilt_sector"))

product |> 
  print(n = Inf)
#> # A tibble: 9 × 28
#>   companies_id           activity_uuid_product_uuid           transition_risk_score transition_risk_low_threshold transition_risk_high_threshold transition_risk_category benchmark_tr_score        profile_ranking reduction_targets country main_activity ep_product                  matched_activity_name                                         matched_reference_product                          unit  co2e_lower co2e_upper emission_profile benchmark   tilt_sector  tilt_subsector           sector_profile scenario  year amount_of_distinct_products equal_weight best_case worst_case
#>   <chr>                  <chr>                                                <dbl>                         <dbl>                          <dbl> <chr>                    <chr>                               <dbl>             <dbl> <chr>   <chr>         <chr>                       <chr>                                                         <chr>                                              <chr>      <dbl>      <dbl> <chr>            <chr>       <chr>        <chr>                    <chr>          <chr>    <dbl>                       <int>        <dbl>     <dbl>      <dbl>
#> 1 nonphilosophical_llama 76269c17-78d6-420b-991a-aa38c51b45b7                 0.59                          0.366                          0.492 high                     1.5C RPS_2030_all                   1                  0.18 germany distributor   tent                        market for shed, large, wood, non-insulated, fire-unprotected shed, large, wood, non-insulated, fire-unprotected m2        45.5      767.    high             all         construction construction residential medium         1.5C RPS  2030                           3        0.333       0            1
#> 2 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.212                         0.366                          0.492 low                      1.5C RPS_2030_all                   0.333              0.09 germany distributor   surface finishing, galvanic market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg        -0.106      1.91  low              all         metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0
#> 3 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.212                         0.366                          0.492 low                      1.5C RPS_2030_all                   0.333              0.09 germany distributor   surface engineering         market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg        -0.106      1.91  low              all         metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0
#> 4 nonphilosophical_llama 76269c17-78d6-420b-991a-aa38c51b45b7                 0.59                          0.575                          0.597 medium                   1.5C RPS_2030_isic_4digit           1                  0.18 germany distributor   tent                        market for shed, large, wood, non-insulated, fire-unprotected shed, large, wood, non-insulated, fire-unprotected m2       269.       355.    high             isic_4digit construction construction residential medium         1.5C RPS  2030                           3        0.333       0            1
#> 5 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.545                         0.575                          0.597 low                      1.5C RPS_2030_isic_4digit           1                  0.09 germany distributor   surface finishing, galvanic market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg        -0.395      1.96  high             isic_4digit metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0
#> 6 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.545                         0.575                          0.597 low                      1.5C RPS_2030_isic_4digit           1                  0.09 germany distributor   surface engineering         market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg        -0.395      1.96  high             isic_4digit metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0
#> 7 nonphilosophical_llama 76269c17-78d6-420b-991a-aa38c51b45b7                 0.59                          0.492                          0.597 medium                   1.5C RPS_2030_tilt_sector           1                  0.18 germany distributor   tent                        market for shed, large, wood, non-insulated, fire-unprotected shed, large, wood, non-insulated, fire-unprotected m2        87.7      787.    high             tilt_sector construction construction residential medium         1.5C RPS  2030                           3        0.333       0            1
#> 8 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.295                         0.492                          0.597 low                      1.5C RPS_2030_tilt_sector           0.5                0.09 germany distributor   surface finishing, galvanic market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg         0.491      0.949 medium           tilt_sector metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0
#> 9 nonphilosophical_llama 833caa78-30df-4374-900f-7f88ab44075b                 0.295                         0.492                          0.597 low                      1.5C RPS_2030_tilt_sector           0.5                0.09 germany distributor   surface engineering         market for deep drawing, steel, 10000 kN press, automode      deep drawing, steel, 10000 kN press, automode      kg         0.491      0.949 medium           tilt_sector metals       other metals             low            1.5C RPS  2030                           3        0.333       0.5          0

company <- output |> 
  unnest_company() |>
  filter(companies_id == "nonphilosophical_llama",
         benchmark_tr_score_avg %in% c("1.5C RPS_2030_all", "1.5C RPS_2030_isic_4digit", "1.5C RPS_2030_tilt_sector"))

company |> 
  print(n = Inf)
#> # A tibble: 12 × 15
#>    companies_id           country main_activity benchmark_tr_score_avg    benchmark   emission_profile emission_profile_share profile_ranking_avg co2_avg scenario year  reduction_targets_avg transition_risk_score_avg transition_risk_category transition_risk_category_share
#>    <chr>                  <chr>   <chr>         <chr>                     <chr>       <chr>                             <dbl>               <dbl>   <dbl> <chr>    <chr>                 <dbl>                     <dbl> <chr>                                             <dbl>
#>  1 nonphilosophical_llama germany distributor   1.5C RPS_2030_all         all         high                              0.333               0.556    101. 1.5C RPS 2030                   0.12                     0.338 high                                              0.333
#>  2 nonphilosophical_llama germany distributor   1.5C RPS_2030_all         all         medium                            0                   0.556    101. 1.5C RPS 2030                   0.12                     0.338 medium                                            0    
#>  3 nonphilosophical_llama germany distributor   1.5C RPS_2030_all         all         low                               0.667               0.556    101. 1.5C RPS 2030                   0.12                     0.338 low                                               0.667
#>  4 nonphilosophical_llama germany distributor   1.5C RPS_2030_all         all         <NA>                              0                   0.556    101. 1.5C RPS 2030                   0.12                     0.338 <NA>                                              0    
#>  5 nonphilosophical_llama germany distributor   1.5C RPS_2030_isic_4digit isic_4digit high                              1                   1        101. 1.5C RPS 2030                   0.12                     0.56  high                                              0    
#>  6 nonphilosophical_llama germany distributor   1.5C RPS_2030_isic_4digit isic_4digit medium                            0                   1        101. 1.5C RPS 2030                   0.12                     0.56  medium                                            0.333
#>  7 nonphilosophical_llama germany distributor   1.5C RPS_2030_isic_4digit isic_4digit low                               0                   1        101. 1.5C RPS 2030                   0.12                     0.56  low                                               0.667
#>  8 nonphilosophical_llama germany distributor   1.5C RPS_2030_isic_4digit isic_4digit <NA>                              0                   1        101. 1.5C RPS 2030                   0.12                     0.56  <NA>                                              0    
#>  9 nonphilosophical_llama germany distributor   1.5C RPS_2030_tilt_sector tilt_sector high                              0.333               0.667    101. 1.5C RPS 2030                   0.12                     0.393 high                                              0    
#> 10 nonphilosophical_llama germany distributor   1.5C RPS_2030_tilt_sector tilt_sector medium                            0.667               0.667    101. 1.5C RPS 2030                   0.12                     0.393 medium                                            0.333
#> 11 nonphilosophical_llama germany distributor   1.5C RPS_2030_tilt_sector tilt_sector low                               0                   0.667    101. 1.5C RPS 2030                   0.12                     0.393 low                                               0.667
#> 12 nonphilosophical_llama germany distributor   1.5C RPS_2030_tilt_sector tilt_sector <NA>                              0                   0.667    101. 1.5C RPS 2030                   0.12                     0.393 <NA>                                              0
```

<sup>Created on 2024-06-17 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

----

TODO

- [x] [Link related issue/PR]([url](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). 
- [x] Describe the goal of the PR. Avoid details that are clear in the diff.
- [x] Mark the PR as draft.
- [x] [Include a unit test](https://code-review.tidyverse.org/reviewer/aspects.html#sec-tests).
- [x] Review your own PR in "Files changed".
- [x] Ensure the PR branch is updated.
- [x] Ensure the checks pass.
- [x] Change the status from draft to ready.
- [x] Polish the PR description to reflect what it ended up doing.
- [x] Polish the PR title as you'd like others to read it from the git log.
- [x] Assign a reviewer.
- [ ] Document user-facing changes in the changelog.

### CODE REVIEW

File: R/add_transition_risk_category_at_company_level.R [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    insert_row_with_na_in_risk_category() |>
+    rename(benchmark_tr_score_avg = "grouped_by", transition_risk_category = "risk_category", transition_risk_category_share = "value")
+}
+
+join_risk_categories_at_company_level <- function(data, risk_categories) {
+  data |>
+    mutate(original_emission_profile = .data$emission_profile) |>
+    left_join(risk_categories, by = c("companies_id", "benchmark_tr_score_avg", "emission_profile" = "transition_risk_category")) |>
+    rename(transition_risk_category = "original_emission_profile")
+}
```
Comment (kalashsinghal) [inline R/add_transition_risk_category_at_company_level.R:14]: @maurolepore I haven't shifted this function to `tiltTransitionRisk` because we don't have the company level dataset of transition risk profile in `tiltTransitionRisk`, however I would be happy to shift these functions to `tiltTransitionRisk` if you think otherwise. :)

FYI: We could create a separate ticket which will shift all the functions related to tranistion risk profile indicator involving `score_transition_risk` and `score_transition_risk_polish` to the repo `tiltTransitionRisk`.

**Comment by Tilmon** [comment]
@kalashsinghal thanks, looks good to me!

File: R/add_transition_risk_category_at_company_level.R [hunk 2]
```diff
@@ -0,0 +1,14 @@
+add_transition_risk_category_at_company_level <- function(data) {
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:1]: If we  have `add_transition_risk_category_at_company_level(), then do we need to rename `add_transition_risk_category()` to `add_transition_risk_category_at_product_level()`?

File: R/add_transition_risk_category_at_company_level.R [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  data |>
+    rename(grouped_by = "benchmark_tr_score", risk_category = "transition_risk_category") |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    rename(benchmark_tr_score_avg = "grouped_by", transition_risk_category = "risk_category", transition_risk_category_share = "value")
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:6]: Maybe the calls to `rename()` should move to an `adapt_*()` of some kind. The task of calculating risk categories is higher-level and different to the lower-level task of adapting the data from one format to the other (i.e. renaming column names in this case).

File: R/add_transition_risk_category_at_company_level.R [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+join_risk_categories_at_company_level <- function(data, risk_categories) {
+  data |>
+    mutate(original_emission_profile = .data$emission_profile) |>
+    left_join(risk_categories, by = c("companies_id", "benchmark_tr_score_avg", "emission_profile" = "transition_risk_category")) |>
+    rename(transition_risk_category = "original_emission_profile")
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:13]: Similar to above, here we seem to be mixing multiple concerns. Maybe the ranaming tasks should move to adaptors.

File: R/add_transition_risk_category_at_company_level.R [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    rename(grouped_by = "benchmark_tr_score", risk_category = "transition_risk_category") |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    rename(benchmark_tr_score_avg = "grouped_by", transition_risk_category = "risk_category", transition_risk_category_share = "value")
+}
+
+join_risk_categories_at_company_level <- function(data, risk_categories) {
+  data |>
+    mutate(original_emission_profile = .data$emission_profile) |>
+    left_join(risk_categories, by = c("companies_id", "benchmark_tr_score_avg", "emission_profile" = "transition_risk_category")) |>
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:12]: I see lines overflowing 80 characters (tidyverse style guide).

File: R/add_transition_risk_category_at_company_level.R (hunk 1 above)
Comment (maurolepore) [reply R/add_transition_risk_category_at_company_level.R:14]: I fail to follow the argument but I think it's fine. The architecture is structured to serve your purposes. Use it however it makes your work easier to do and maintain in the long term.

While I don't quite understand the comment, my intuition suggests that if you want to move a function to a higher-level component (tiltTransitionRisk) but you can't because the function mentions lower-level concepts (column names) that don't exist in the higher-level component, then likely this means that the function is mixing code at different levels of abstraction that may belong to separate functions. Above I noted some code that may move to an adaptor and that may solve the problem.

File: R/transition_risk_profile.R [hunk 6]
```diff
@@ -101,8 +101,11 @@ transition_risk_profile <- function(emissions_profile,
     polish_best_case_worst_case() |>
     polish_transition_risk_at_product_level()
 
+  risk_categories_at_company_level <- add_transition_risk_category_at_company_level(product)
+
   company <- transition_risk_scores |>
     unnest_company() |>
+    join_risk_categories_at_company_level(risk_categories_at_company_level) |>
     polish_transition_risk_at_company_level()
```
Comment (maurolepore) [inline R/transition_risk_profile.R:109]: It would be nice to structure this chunk of code so that we create `product` and `company` with similar pipes. That may read more easily, but also may have the beneficial side effect of forcing us to find similarities that may result in more reusable code.

File: tests/testthat/test-transition_risk_profile.R [hunk 7]
```diff
@@ -103,3 +103,56 @@ test_that("outputs `NA` transition risk category for `NA` transition risk score
   # `transition_risk_category` is `NA` for `NA` transition risk score
   expect_true(is.na(unique(output$transition_risk_category)))
 })
+
+test_that("outputs columns `transition_risk_category_share` and `transition_risk_category` at company level", {
```
Comment (maurolepore) [inline tests/testthat/test-transition_risk_profile.R:107]: A few things:

* This test doesn't seem to need a seed. We're only testing for column names -- not their content.
* The setup is really large and burries the intention of the test.
* Are we planning more tests? The fact that the output has a specific column doesn't say much about whether or not the values of that column are correct. 
* What reasaonable edge cases do w need to test?

**Review by maurolepore — COMMENTED** [review]
Thanks @kalashsinghal 

I did a pretty broad review. Hopefully it would give you some ideas. Feel free to disregard them. The tight timeline for this indicator implicitly means that we're collectively expecting less robust code than we expected for other indicators. So you'll need to engage with Tilman/Anne and judge for yourselves when it's good enough for now.

File: R/add_transition_risk_category_at_company_level.R (hunk 2 above)
Comment (kalashsinghal) [reply R/add_transition_risk_category_at_company_level.R:1]: Right! I created a separate issue for this work: https://github.com/2DegreesInvesting/tiltTransitionRisk/issues/22

File: tests/testthat/test-transition_risk_profile.R (hunk 7 above)
Comment (kalashsinghal) [reply tests/testthat/test-transition_risk_profile.R:107]: @maurolepore The edge cases are same for `add_transition_risk_category_at_company_level` function because it uses the function `epa_at_company_level()` from tiltIndicator package. I can see a lot tests for this function in tiltIndicator which covers possible edge case. Do you want me to copy all those test cases here as well?

File: R/add_transition_risk_category_at_company_level.R (hunk 3 above)
Comment (kalashsinghal) [reply R/add_transition_risk_category_at_company_level.R]: Thanks! refactored :)

File: R/add_transition_risk_category_at_company_level.R (hunk 4 above)
Comment (kalashsinghal) [reply R/add_transition_risk_category_at_company_level.R]: Refactored :)

File: R/transition_risk_profile.R (hunk 6 above)
Comment (kalashsinghal) [reply R/transition_risk_profile.R]: @maurolepore Please review the code again to see if it meets this expectation of yours! Thanks!

File: R/add_transition_risk_category_at_company_level.R (hunk 1 above)
Comment (kalashsinghal) [reply R/add_transition_risk_category_at_company_level.R]: Adaptors won't solve this issue. This issue can be solved if all the lower level functions are moved to the higher level component. Should I create a separate ticket for this work?

File: tests/testthat/test-transition_risk_profile.R (hunk 7 above)
Comment (maurolepore) [reply tests/testthat/test-transition_risk_profile.R:107]: Mmm it shouldn't be about what I "want". Please take my reviews as suggestions intended to spark ideas about how to improve the code. As the author of this function and as the maintainer of the package, you are the one with the deepest context and you're welcome to take or leave my ideas -- which due to my superficial understanding may well be off.

Comment (kalashsinghal) [reply tests/testthat/test-transition_risk_profile.R:107]: @maurolepore For example, we have column A and column B which have similar unique values but belongs to different tables, then should I test the column A if we have good amount of tests for column B? Is it a good practice to have similar tests for column A as well?

File: R/add_transition_risk_category_at_company_level.R [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  tilt_profile(nest_levels(product, company))
+}
+
+join_risk_categories_at_company_level <- function(data, risk_categories) {
+  data |>
+    create_extra_emission_profile_col() |>
+    left_join(risk_categories, by = c("companies_id",
+      "benchmark_tr_score_avg",
+      "emission_profile" = "transition_risk_category"
+    )) |>
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:24]: > Strive to limit your code to 80 characters per line. ...If a function call is too long to fit on a single line, use one line each for the function name, each argument, and the closing ). This makes the code easier to read and to change later.

https://style.tidyverse.org/syntax.html#long-lines

File: R/transition_risk_profile.R [hunk 9]
```diff
@@ -79,9 +79,26 @@ transition_risk_profile <- function(emissions_profile,
...
+  ) |>
+    add_transition_risk_category_at_company_level()
+}
+
+transition_risk_profile_impl <- function(emissions_profile,
+                                    sector_profile,
+                                    co2,
+                                    all_activities_scenario_sectors,
+                                    scenarios,
+                                    pivot_wider = FALSE) {
```
Comment (maurolepore) [inline R/transition_risk_profile.R:98]: `usethis::use_tidy_style()`

File: R/add_transition_risk_category_at_company_level.R [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    adapt_tr_product_cols_to_tiltIndicator_cols() |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    adapt_tiltIndicator_cols_to_tr_company_cols()
+
+  company <- data |>
+    unnest_company() |>
+    join_risk_categories_at_company_level(risk_categories)
+
+  tilt_profile(nest_levels(product, company))
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:15]: This seems to belong to `profile_transition_risk_impl()`

File: R/add_transition_risk_category_at_company_level.R [hunk 11]
```diff
@@ -0,0 +1,49 @@
+add_transition_risk_category_at_company_level <- function(data) {
+  product <- data |>
+    unnest_product()
+
+  risk_categories <- product |>
+    adapt_tr_product_cols_to_tiltIndicator_cols() |>
+    epa_at_company_level() |>
+    insert_row_with_na_in_risk_category() |>
+    adapt_tiltIndicator_cols_to_tr_company_cols()
```
Comment (maurolepore) [inline R/add_transition_risk_category_at_company_level.R:9]: Looks better. At lease the function doesn't mention column names so keeps the code at a higher level of abstraction. 

A further enhancement may be to move the calls to `adapt*()` outside the body of the function. But that may be an overkill at this stage. 

This idea comes from noticing that `add_*()` functions typically only add columns or rows (e.g. `tibble::add_column()`, `tibble::add_row()`) or -- more closely to what we're doing here -- compute some summary and then add it as a column (e.g. `dplyr::add_count()`). With this approach, the adaptors don't quite belong into the body of the `add_*()` function. Instead we could apply the adaptors to the `data` before/after we call `add_*()`.

**Review by maurolepore — APPROVED** [review]
Thanks @kalashsinghal LGTM

File: tests/testthat/test-transition_risk_profile.R (hunk 7 above)
Comment (kalashsinghal) [reply tests/testthat/test-transition_risk_profile.R:107]: Ticket: https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/267

File: R/add_transition_risk_category_at_company_level.R (hunk 1 above)
Comment (kalashsinghal) [reply R/add_transition_risk_category_at_company_level.R]: Ticket: https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/268
